### PR TITLE
feat: add stub backend

### PR DESCRIPTION
**Pull request by ServiceDog**

### CODE REVIEW

File: server/src/postgis/aircraft.rs [hunk 1]
```diff
@@ -137,10 +137,10 @@ pub async fn update_aircraft_position(
         .map(|craft| {
             format!(
                 "SELECT arrow.update_aircraft_position(
```
Comment (owlot) [inline server/src/postgis/aircraft.rs:139]: Is there a reason you're not using a prepared statement here? Passing raw values is usually a security risk and prepared statements are cachable if you save the prepared handle when you run it multiple times like this.

Comment (ServiceDog) [reply server/src/postgis/aircraft.rs:139]: Will update

Comment (ServiceDog) [reply server/src/postgis/aircraft.rs:139]: So would something like this be better? 
```rust
    let mut commands = vec![
        "PREPARE update_aircraft_position (
            UUID, GEOM, FLOAT, VARCHAR, TIMESTAMPTZ
        ) AS
        SELECT arrow.update_aircraft_position(
            $1,
            $2,
            $3,
            $4,
            $5
        )".to_string(),
    ];

    for craft in &aircraft {
        commands.push(format!(
            "EXECUTE update_aircraft_position(
                {},
                '{}',
                {},
                '{}',
                '{}'::TIMESTAMPTZ
            )",
            match &craft.uuid {
                Some(uuid) => format!("'{}'::UUID", uuid),
                None => "NULL".to_string(),
            },
            craft.geom,
            craft.altitude_meters,
            craft.callsign,
            craft.time
        ));
    }

    match execute_transaction(commands, pool).await {
        Ok(_) => (),
        Err(_) => {
            postgis_error!("(postgis update_aircraft) Error executing transaction.");
            return Err(AircraftError::Unknown);
        }
    }
```

Not sure if this is avoiding passing raw values

Comment (owlot) [reply server/src/postgis/aircraft.rs:139]: You will need to construct a params vec which you can pass to the execute command. I've never used a PREPARE statement like this on the server side before. You can do this from the client, and that's where the prepared statement will be cached for you as well.

I'm using it for all the queries in svc-storage.

For you, it would be something like this:
```
let mut client = pool.get().await?;
let transaction = client.transaction().await?;
let stmt = transaction.prepare_cached("SELECT arrow.update_aircraft_position($1, $2, $3, $4, $5::TIMESTAMPTZ)").await?;

for craft in &aircraft {
    transaction.execute(&stmt, &[
        match &craft.uuid {
            Some(uuid) => &format!("'{}'::UUID", uuid),
            None => "NULL",
        },
        &craft.geom,
        &craft.altitude_meters,
        &craft.callsign,
        &craft.time
    ]).await?;
}

transaction.commit().await?;
```

Heavily untested, but hopefully you get the idea.
Also see the deadpool_postgres docs where they use prepared statements:
https://docs.rs/deadpool-postgres/latest/deadpool_postgres/
And the postgres module itself:
https://docs.rs/postgres/latest/postgres/index.html

Basically, you always want to use placeholders when working with values in a database.
Even without the use of a prepared statement, you want to pass them as separate params. This eliminates the possibility of someone passing a '; delete from user;' string which will be executed as is, since it'll just be contaminated into the query string.
With parameters, the database engine knows it should handle it as a value and not as a raw query part.

Comment (ServiceDog) [reply server/src/postgis/aircraft.rs]: Alright, updated aircraft position, waypoints, vertiports, and nofly zones to use this method! Nice because I was able to use specific types instead of Strings to represent geometries

https://github.com/Arrow-air/svc-gis/pull/9/commits/c20b4984cd343a9ceec9002b5b41acd44271193b

**Review by owlot — APPROVED** [review]
